pallets/fragments/src/lib.rs

```diff
@@ -46,17 +82,31 @@ pub struct UniqueOptions {
 	pub mutable: bool,
 }
 
-/// **Struct** of a **Fragment**
+/// **Struct** of a **Fragment Definition**
+///
+/// Notes from Giovanni:
+/// 
+/// Fragments definitions are the DNA of fragments.
+///
+/// This is the way to program fragments instances' distribution, expiration and starting permissions
+/// before even creating any fragment instances yet.
+///
+/// #### Remarks
+///
+/// * #immutable - once created there is no way to edit, intentionally.
 #[derive(Encode, Decode, Clone, scale_info::TypeInfo, Debug, PartialEq)]
-pub struct FragmentClass<TFungibleAsset, TAccountId, TBlockNum> {
+pub struct FragmentDefinition<TFungibleAsset, TAccountId, TBlockNum> {
 	/// **Proto-Fragment used** to **create** the **Fragment**
 	pub proto_hash: Hash256,
 	/// ***FragmentMetadata* Struct** (the **struct** contains the **Fragment's name**, among other things)
 	pub metadata: FragmentMetadata<TFungibleAsset>,
-	/// The next owner permissions
+	/// **Actions** (encapsulated in a `FragmentPerms` bitflag enum) that are **allowed to be done** to 
+	/// **any Fragment Instance** that is **created from** the **Fragment Definition** (e.g edit, transfer etc.)
```

Comment:
Actually this just just permission of first sale, next owner. Not transfer etc

Reply:
you mean it'll be **the permissions of any fragment instance created using this Fragment Definition**, correct?

Reply:
nope, just editions, `give` might change permissions

Reply:
oh you're right. i understand how people could be confused. 

i'll mention that `give` might change permissions in the future

Reply:
Giovanni (@sinkingsugar), is it clear enough now? Thanks

Reply:
LGTM 👍